_posts/2020-09-04-meet-the-team.md

```diff
@@ -1,174 +1,198 @@
+#### What do you like to do in your free time?
 
-### What is your role in the Solidity team? On which parts of Solidity do you work the most?
+I enjoy finding security issues with popular open-source software implementations and blogging about my experiences on my personal blog over at https://bshastry.github.io/blog. I also enjoy tapping the closest piece of furniture to the beat of electronic music that is being played.
 
-*In the Solidity team, my role is to help the team find bugs and security issues as soon as possible, preferably before an upcoming release. More specifically, I develop semantic fuzzers for different compiler sub-systems e.g., compiler frontend, optimiser etc.*
+- - -
 
-### What do you like to do in your free time?
+### Chris - [@chriseth](https://github.com/chriseth)
 
-*I enjoy finding security issues with popular open-source software implementations and blogging about my experiences on my personal blog over at https://bshastry.github.io/blog. I also enjoy tapping the closest piece of furniture to the beat of electronic music that is being played.*
+#### What is your professional background?
 
-## Chris - [@chriseth](https://github.com/chriseth)
+During my studies, I have worked on different small open source projects related to Wikipedia and biodiversity. After my PhD in Computational Complexity Theory, where I worked on multi-objective optimization and structural complexity theory, I gathered some experience in the classical closed-source software industry. In 2014, I started the Solidity project and really enjoy the open collaboration of the professional free software world that is very similar in spirit to the research community.
 
-### What is your professional background?
+#### What is your role in the Solidity team? On which parts of Solidity do you work the most?
 
-*During my studies, I have worked on different small open source projects related to Wikipedia and biodiversity. After my PhD in Computational Complexity Theory, where I worked on multi-objective optimization and structural complexity theory, I gathered some experience in the classical closed-source software industry. In 2014, I started the Solidity project and really enjoy the open collaboration of the professional free software world that is very similar in spirit to the research community.*
+With help from Alex Beregszaszi I'm leading the Solidity team. This means that I'm doing a lot of code review and design work, but there is also enough time to write code. My favourite part of the project is the Yul optimizer because you can make a lot of impact with tiny but well-though-out components.
 
-### What is your role in the Solidity team? On which parts of Solidity do you work the most?
+#### What would you like to see in Solidity in the future? (focus on Solidity)
```

Comment:
```suggestion
#### What would you like to see in Solidity in the future?
```
That was rather a hint for us than part of the actually section header, wasn't it :-)?

Reply:
Oh yes it was!